CHANGELOG.md

```diff
@@ -4,20 +4,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This change log adheres to standards from [Keep a CHANGELOG](http://keepachangelog.com).
 
 ## [Unreleased]
-
-
-## [2.8.0] - 2017-10-18
```

Comment:
I think this needs rebasing, and some fixing - it appears some info has been lost

Reply:
Ooops, sorry - fixed! If you'd like, I can drop the changes to CHANGELOG completely. Not sure if it's supposed to be part of a PR.

Reply:
nah it's good to add it to "unreleased"